### PR TITLE
feat: adds ticket invite key env to build

### PR DESCRIPTION
**Pull request by mikaelbr**
Adds env to store invite key. Should be in a secure vault, but this isn't actually very secret sensitive stuff.

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -11,4 +11,5 @@ COPY package*.json ./
 RUN npm install --production
 
 ENV NODE_ENV=production
+ENV TICKET_INVITE_KEY=$ticket_invite_key
```
Comment (torfjor) [inline Dockerfile:14]: Should the build system be aware of the invite key? If not, it's enough to grab the variable from the environment at runtime.

```shell-script
$ gcloud run services update --update-env-vars=TICKET_INVITE_KEY=XXX SERVICE_NAME
```

Environment variables configured for a service persist between deployments unless explicitly cleared or updated.

Comment (mikaelbr) [reply Dockerfile:14]: It has to exist when running `node` inside the docker image. If `--update-env-vars` on cloud run does that, it works for me!

Comment (torfjor) [reply Dockerfile:14]: Yes, all configured environment variables are passed down to the container at runtime.

Comment (mikaelbr) [reply Dockerfile:14]: Where should this be, then? Is it just setting variables on the cloud run instance, or do we need to explicitly run the command in say the cloudbuild file?

In other words: Can I just remove all docker/cloudrun environment variable management and just assume it exist in runtime?

Comment (torfjor) [reply Dockerfile:14]: Yes.

**Comment by mikaelbr** [comment]
@torfjor Sounds almost too good to be true with envs in Docker, but if you say so 😄  Removed all gcp/docker env code now

**Pull request merged**